connectors/Cloudmersive PDF/apiDefinition.swagger.json

```diff
@@ -0,0 +1,5001 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "v1",
+        "title": "Cloudmersive PDF",
+        "description": "PDF lets you effortlessly create, convert, process, edit, work with, encrypt, and sign PDF files."
+    },
```

Comment:
Please add api annotation back to the the info.
```
,
    "x-ms-api-annotation": {
      "status": "Preview"
    }
```

Reply:
Added x-ms-api-annotation per your request